### ISSUE TITLE
Bundler can't resolve dependecies (or can it?)

### ISSUE THREAD

**Issue by d-vine** [TheWidlarzGroup/react-native-video#80]
When running the bundler I get the following output:

```
Unable to resolve module ReactNativeViewAttributes from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/Video.ios.js
Unable to resolve module NativeModules from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/Video.ios.js
Unable to resolve module StyleSheet from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/Video.ios.js
Unable to resolve module ReactPropTypes from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/Video.ios.js
Unable to resolve module StyleSheetPropType from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/Video.ios.js
Unable to resolve module NativeMethodsMixin from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/Video.ios.js
Unable to resolve module flattenStyle from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/Video.ios.js
Unable to resolve module merge from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/Video.ios.js
Unable to resolve module deepDiffer from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/Video.ios.js
Unable to resolve module keyMirror from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/VideoResizeMode.js
Unable to resolve module ViewStylePropTypes from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/VideoStylePropTypes.js
Unable to resolve module ReactPropTypes from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/VideoStylePropTypes.js
Unable to resolve module keyMirror from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/VideoResizeMode.js
Unable to resolve module ViewStylePropTypes from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/VideoStylePropTypes.js
Unable to resolve module ReactPropTypes from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/VideoStylePropTypes.js
Unable to resolve module ReactNativeViewAttributes from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/Video.ios.js
Unable to resolve module NativeModules from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/Video.ios.js
Unable to resolve module StyleSheet from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/Video.ios.js
Unable to resolve module ReactPropTypes from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/Video.ios.js
Unable to resolve module StyleSheetPropType from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/Video.ios.js
Unable to resolve module NativeMethodsMixin from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/Video.ios.js
Unable to resolve module flattenStyle from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/Video.ios.js
Unable to resolve module merge from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/Video.ios.js
Unable to resolve module deepDiffer from /Users/dvine/dev/goldfinger/iphone/node_modules/react-native-video/Video.ios.js
```

The app seems to work, though. But I get unpredictable crashes of the same codebase on different (but identical) targets (dev, prod). Not sure if they are related.

**Comment by d-vine** [comment]
The same effect in a different component is mentioned here:
https://github.com/facebook/react-native/issues/1808#issuecomment-117258658
But I'm still none the wiser what to do about it.

**Comment by brentvatne** [comment]
Related to: https://github.com/brentvatne/react-native-video/pull/71

Will try to fix this for you today

**Comment by d-vine** [comment]
Great! Thank you! But this is nothing that I want to ruin your weekend with.

**Comment by PhilippKrone** [comment]
@brentvatne That would be really awesome. Is it possible that reloading after javascript code changes takes longer if those warnings are thrown? Or is this just a feeling?

**Comment by brentvatne** [comment]
Hi all, this is fixed with react-native-video 0.5.0, which I tested by upgrading the example app to react-native 0.8.0, no more module errors being shown on my end. Let me know if this doesn't solve the problem for you!
